### PR TITLE
BAVL-245: Add new event for video appointments cancelled

### PR DESCRIPTION
**Pull request by timharrison-moj**

### CODE REVIEW

**Review by mxco86 — APPROVED** [review]

**Review by mxco86 — DISMISSED** [review]

File: spec/schemas/prison-offender-events/prisoner.video-appointment.cancelled.yaml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+name: prison-offender-events.video-appointment.cancelled
```
Comment (petergphillips) [inline spec/schemas/prison-offender-events/prisoner.video-appointment.cancelled.yaml:1]: think this should be `prison-offender-events.prisoner.video-appointment.cancelled`

Comment (timharrison-moj) [reply spec/schemas/prison-offender-events/prisoner.video-appointment.cancelled.yaml:1]: I was aiming for `prison-offender-events.video-appointment.cancelled` - but think I've left a `prisoner` in one place by mistake.

Comment (timharrison-moj) [reply spec/schemas/prison-offender-events/prisoner.video-appointment.cancelled.yaml]: Now updated to be consistent event name.

Comment (timharrison-moj) [reply spec/schemas/prison-offender-events/prisoner.video-appointment.cancelled.yaml]: Updated to recommended name including `prisoner`

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**